.github/ISSUE_TEMPLATE/feature_template.md

```diff
@@ -0,0 +1,17 @@
+---
+labels: web-ui
```

Comment:
Maybe we could add the proposal label here too https://github.com/sensu/web/labels/issue%3A%20proposal